app/storage/encrypted_questionnaire_storage.py

```diff
@@ -1,4 +1,5 @@
 import hashlib
+import simplejson
```

Comment:
Not directly related to this review as we started doing this elsewhere *but*; I read [the package documentation](https://pypi.python.org/pypi/simplejson/) and I *think* `simplejson` should be the same as the `json` package that's part of the standard library.

Begs the question as to why we were seeing different behaviour with decimals (I think) but probably worth checking before we go and embed simplejson as a dependency everywhere.